src/components/feed/list/FeedCard.tsx

```
@@ -248,8 +249,8 @@ const renderContent = (content: string) => {
     displayText = content.slice(0, 140) + '...';
     isLong = true;
   }
-
-  const parsed = parseMentionsToJSX(displayText);
+  const router = useRouter();
```

ESLint

[error] React Hook "useRouter" is called in function "renderContent" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)